nailgun/entities.py

```diff
@@ -1579,6 +1632,11 @@ def create_missing(self):
                  |-> domain
                  '-> environment
 
+        In case, you have full set of instance attributes that is necessary to
+        pass, don't forget to put False value for your 'create()' method call,
+        so flow will not enter current 'create_missing' method and not throw
+        an exception
+
```

Comment:
I have no idea what that method is doing. Basically, just wanted to emphasize on thing that took few hours for me to figure out. (to avoid mentioned exception), but you are right, it will be better to do something with that ```configure_nailgun()``` method